process/tac-onboarding.md

```diff
@@ -0,0 +1,128 @@
+# OpenSSF TAC Onboarding
+
+Welcome to the OpenSSF Technical Advisory Council. Thank you in advance for your dedication to the work this role will entail.
+
+Per the [OpenSSF charter][charter] 5c:
+
+> The role of the TAC is to structure and facilitate collaboration among the Technical Initiatives.
+>
+> The TAC will be responsible for:
+>
+> i. developing an overall technical vision for the community;
+>
+> ii. approving, establishing, structuring, organizing, and archiving Technical Initiatives;
+>
+> iii. creating, maintaining and amending the policies and procedures for Technical Initiative onboarding and communications with the TAC;
+>
+> iv. working with the Technical Initiatives to identify any resource or funding requirements and prioritizing recommendations to the Governing Board;
+>
+> v. facilitating crowdfunding opportunities in support of OpenSSF Technical Initiatives;
+>
+> vi. annually electing a chairperson to preside over meetings, set the agenda for meetings, ensure meeting minutes are taken, and who will also serve on the Governing Board as the TAC's representative (the "TAC Representative"); and
+>
+> vii. coordinating such other technical community matters related to the success of Technical Initiatives and the mission of the OpenSSF
+
+Please take a few moments to familiarize yourself with the [charter][charter].
+
+This onboarding document is designed to provide some introductory and ***non-exhaustive*** information to new TAC members.
+As a TAC member, it is your responsibility to keep this document up to date.
+
+By design, this onboarding document contains limited information to set expectations and provide a TAC member with generalities about the role and work we do. For specific instructions, guidance, templates, etc., please refer to the content in the [`/process` directory](./README.md).
+
+It covers details on topics such as how we make decisions, technical initiative lifecycles, and the funding process.
+
+## Open questions
+
+- Are there private mailing lists?
+- Are there additional access requirements not accounted for in this document?
+- What are our moderation responsibilities and privileges (GitHub, mailing lists, Zoom)?
```

Comment:
I'm not sure we ever created a moderator schedule. As far as privileges are concerned, I believe we can open a meeting, etc. I believe only staff can create repos, mailing lists, etc.